frontend/copr-frontend.spec

```diff
@@ -296,16 +296,22 @@ EOF
 %py_byte_compile %{__python3} %{buildroot}%{_datadir}/copr/coprs_frontend/coprs
 %py_byte_compile %{__python3} %{buildroot}%{_datadir}/copr/coprs_frontend/alembic
 
+%if 0%{?fedora} && 0%{?fedora} > 41
```

Comment:
Are you sure we have to special-case F40?  If yes, Mock on F40 is broken :shrug:

Reply:
I think this adresses https://src.fedoraproject.org/rpms/copr-frontend/pull-request/3#comment-252345 ?

Reply:
Aha, good to document at least!  Anyway, in practice we do not use locked users.  We do things like `su - copr-fe` on a daily basis.

Reply:
I would keep the copr-fe user (and other PRs too) unlocked.  If we disagree on this, we can do `sed -i 's/u!/u/' copr-frontend.conf` if on `%if 0%{?fedora} && 0%{?fedora} <= 41`.

Otherwise, since we only support Fedora, the use of sysusers RPM feature should be unconditional.

Reply:
Why have it unlocked? We have it locked now (see the line with `usermod -L copr-fe`) so this is no modification of current behaviour. You can still `su` to locked account. You just cannot log in (in terminal or ssh) to that account.

Reply:
> Otherwise, since we only support Fedora, the use of sysusers RPM feature should be unconditional.

What about Euler? 

Additionally if we remove this condition that we have to add requires on specific version of `rpm` because on older version of Fedora you can have rpm from GA release that does not support this.

Reply:
> Why have it unlocked? 

Because that's what we practically do (we create users deterministically via ansible, and execute e.g. `copr-frontend create-chroot` manually, etc.).

Reply:
> What about Euler?

Good point, @pkking how far is openEuler with [Rpm >= 4.19 support](https://rpm-software-management.github.io/rpm/manual/users_and_groups.html)?

Reply:
@pkking of course if you create users differently (not via an RPM installation, %post scriptlets, etc.), you don't need to have RPM 4.19 at all - then, whatever we do in this PR can not affect your openEuler deployments; nb. our Copr deployments are neither affected by this change.

Reply:
Hmm, now I read better, sorry - learning something new :sweat_smile: What is the practical effect of `usermod -L`?  I thought it basically does `/sbin/nologin`, but as I understand it now - it only ever makes users' passwords non-working (modifying etc/shadow).  If that's what it does, I don't think we need that at all (we don't create users with passwords).

Reply:
> @pkking of course if you create users differently (not via an RPM installation, %post scriptlets, etc.), you don't need to have RPM 4.19 at all - then, whatever we do in this PR can not affect your openEuler deployments; nb. our Copr deployments are neither affected by this change.

Hi, the latest openEuler LTS release is using the rpm `version 4.18`, the next LTS release will be at 2026/03, and yes, we run copr in k8s cluster and create user using a `useradd` command:
```shell
RUN groupadd -r copr-fe -g 997
RUN useradd -r copr-fe -u 997 -g 997 -d /usr/share/copr/coprs_frontend
```